SpatialGDK/Source/SpatialGDKEditor/Public/SpatialTestSettings.h

```diff
@@ -46,6 +46,12 @@ class SPATIALGDKEDITOR_API FSpatialTestSettings
 	// Generated map override config base filename for generated maps applied to specific map, if exists
 	static const FString GeneratedOverrideSettingsBaseFilename;
 
+	// Creates the configuration file pathname for the given MapName.
+	static FString GenerateMapConfigurationFilename(const FString& MapName);
+
+	// Load settings from config file which will override current settings
+	static void Load(const FString& TestSettingOverridesFilename);
```

Comment:
Minor: put this back in its original place to reduce churn.

Reply:
fair enough.

Reply:
Yeah Miron is right, we wanted to make this public